pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go

```diff
@@ -73,8 +73,10 @@ func (f *Makefile) SetTemplateDefaults() error {
 //nolint:lll
 const makefileTemplate = `# Image URL to use all building/pushing image targets
 IMG ?= {{ .Image }}
-# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.30.0
+# K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+# Also it refers to the version of Kubernetes that Kind set up.
+K8S_VERSION = v1.30.0
+K8S_VERSION_TRIMMED_V = $(subst v,,$(K8S_VERSION))
```

Comment:
We tag the ENV TEST version which is compatible, I do not think that we should add the k8s version here. It is not used in any place.

Reply:
Looking a bit forward in my PR it is [used](https://github.com/hiddenmarten/kubebuilder/blob/4fc202a92e34bf5dc511dc593ca165854fb8623a/pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go#L242-L250) in creating kind cluster.

I also attempted to explain this behavior in [a comment above](https://github.com/hiddenmarten/kubebuilder/blob/4fc202a92e34bf5dc511dc593ca165854fb8623a/pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go#L77), but I may not have been entirely clear.

I would greatly appreciate your assistance in making the comment more understandable. Thank you!

Reply:
I agree, let's take that our world isn't ideal and make two variables instead of one :)